packages/hackathon/src/scenes/UsersScene/components/HackerList.tsx

```diff
@@ -81,7 +71,19 @@ export const HackerList: FC<HackerListProps> = ({
   const rows = hackers
     .slice(startIdx, startIdx + PAGE_SIZE)
     .map((hacker, idx) => (
-      <DataTableItem key={idx} id={idx.toString()}>
+      <DataTableItem
+        key={idx}
+        id={idx.toString()}
```

Comment:
See other comment regarding id

Reply:
I'm going to take it as a project to eventually support additional (simple) types besides `string` (`number` seems reasonable at the very least)